frontend/docsettings.lua

```diff
@@ -5,14 +5,25 @@ in the so-called sidecar directory
 ]]
 
 local DataStorage = require("datastorage")
+local Device = require("device")
 local LuaSettings = require("luasettings")
 local dump = require("dump")
 local ffiutil = require("ffi/util")
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
 local util = require("util")
 
-local DocSettings = LuaSettings:extend{}
+local DocSettings = LuaSettings:extend{
+    cover_ext = {
+        png  = true,
+        jpg  = true,
+        jpeg = true,
+        gif  = true,
+        tif  = true,
+        tiff = true,
+        svg  = true,
```

Comment:
If this is to keep it limited, it's probably better to keep it truly limited to png, jpg and svg.

Otherwise, see:
https://github.com/koreader/koreader/blob/af45ec75c3ee0b2315b5086d15dbb8702879836d/frontend/document/pdfdocument.lua#L384-L404